docs/api/browser-window.md

```diff
@@ -1162,6 +1162,18 @@ Returns `boolean` - Whether the window's document has been edited.
 * `opts` Object (optional)
   * `stayHidden` boolean (optional) -  Keep the page hidden instead of visible. Default is `false`.
   * `stayAwake` boolean (optional) -  Keep the system awake instead of allowing it to sleep. Default is `false`.
+  * `outputSize` [Size](structures/size.md) (optional) - The desired width and height in pixels (positive numbers) of the returned image.
+
+Returns `Promise<NativeImage>` - Resolves with a [NativeImage](native-image.md)
+
+Captures a snapshot of the page within `rect`. Omitting `rect` will capture the whole visible page.
+The page is considered visible when its browser window is hidden and the capturer count is non-zero.
+If you would like the page to stay hidden, you should ensure that `stayHidden` is set to true.
```

Comment:
Looks like this got duplicated? It's [here](https://github.com/electron/electron/pull/41281/files#diff-e8a455d5c2bb8c09554011e09a285d6e66c3c681082ae9998f2a4a026eaf0b03R1178) already.

Reply:
Sorry, I missed that. [Fixed in the last commit](https://github.com/electron/electron/pull/41281/commits/861d79b9f12cb9b09055d2580925203110e54bb1)